src/components/Comments/Comment.js

```diff
@@ -397,7 +413,8 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(14),
     color: COLORS.white,
-    marginLeft: 28
+    marginLeft: 20,
+    flex: 1
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style for `marginLeft` and `flex` has been updated. Make sure this change does not affect the layout in an undesired way.
```diff
+     marginLeft: 20,
+     flex: 1
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->